src/common/listActions/ListActions.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Button from '../button/Button';
+import Text from '../text/Text';
+import Select from '../select/Select';
+import styles from './listActions.module.scss';
+
+type ListActionItem<T> = {
+  id: string;
+  label: string;
+  buttonText?: string;
+  onClick?(selectedRows: T[]): void;
+  renderComponent?(selectedRows: T[], resetSelectedRows: () => void): React.ReactNode;
+};
+
+interface ListActionsProps<T> {
+  selectedRows: T[];
+  listActions: ListActionItem<T>[];
+  resetSelectedRows(): void;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const ListActions = <T extends any>({ selectedRows, resetSelectedRows, listActions }: ListActionsProps<T>) => {
```

Comment:
Could `any` be replaced with `unknown` or some `union` type?

Reply:
Done. Now it's "Object or any primitive type".